model/config.go

```diff
@@ -300,6 +300,7 @@ type ServiceSettings struct {
 	DEPRECATED_DO_NOT_USE_RestrictPostDelete          *string `json:"RestrictPostDelete" mapstructure:"RestrictPostDelete"`                   // This field is deprecated and must not be used.
 	DEPRECATED_DO_NOT_USE_AllowEditPost               *string `json:"AllowEditPost" mapstructure:"AllowEditPost"`                             // This field is deprecated and must not be used.
 	PostEditTimeLimit                                 *int
+	ServerFirstRun                                    *int64 `json:"ServerFirstRun"`
```

Comment:
Why is this part of the config and not the DB?

Reply:
My thinking was - we need to start reporting ASAP, maybe even before DB connection is made.
cc @grundleborg  thoughts?

Reply:
I don't think we should be using the config like this. In HA setups where orchestration deploys the config files (e.g. they are kept in Git and deployed by Chef or Ansible or the like), it will get overwritten every time they deploy. This should be in the "Systems" table in the database. I don't think there's any need to send telemetry before the database is up and running.

Reply:
Thanks. Will re-do using the database